maintenance/MaintenanceTask.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class MaintenanceTask extends \Wikia\Tasks\Tasks\BaseTask {
+	public function run( $script ) {
+		return shell_exec( $script );
```

Comment:
Let's validate `$script` - is its path relative to app's root?  If so, let's check if `file_exists( $IP. $script)`